tennis-alliance/src/app/page.tsx

```diff
@@ -82,14 +116,26 @@ export default function Home() {
           target="_blank"
           rel="noopener noreferrer"
         >
-          <h2>
-            Deploy <span>-&gt;</span>
-          </h2>
-          <p>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
+                     By{' '}
+            <Image
+              src="/vercel.svg"
+              alt="Vercel Logo"
+              className={styles.vercelLogo}
+              width={100}
+              height={24}
+              priority
+            />
         </a>
       </div>
+      <div>
+        // The quote 
```

Comment:
This should not appear in the actual site. You can make comments with `{/* A JSX comment */}`.